app/src/main/java/com/github/h3lp3rs/h3lp/MainPageActivity.kt

```diff
@@ -41,20 +132,44 @@ class MainPageActivity : AppCompatActivity() {
                 R.id.nav_home -> findViewById<DrawerLayout>(R.id.drawer_layout).closeDrawer(
                     GravityCompat.START
                 )
-                else -> {
-                    true
-                }
             }
             true
         }
+    }
 
+    /**
+     * Starts activity based on the entered element in the search field.
+     */
+    private fun findActivity(listItem: String, view: View) {
+        when (listItem) {
+            PROFILE -> goToProfileActivity(view)
+            CPR_RATE -> goToCprActivity(view)
+            TUTORIAL -> viewPresentation(view)
+        }
+    }
+
+    /**
+     * Displays a message using snackbar under the horizontal scroll view
+     * @param message message to display
+     */
+    private fun displayMessage(message: String) {
+        val fab = findViewById<View>(R.id.horizontalScrollView)
```

Comment:
Thank you for your comment!

This can actually be used to display any message, so I changed the method's signature to this.

/**
     * Displays a message using snackbar under the view
     * @param message message to display
     * @param view the view under which the message is shown
     */
    private fun displayMessage(message: String,view:View)

For the error message as it is only related to the search bar, I changed its name to displayErrorMessageAfterSearch().

Fab = floating action button, I was using them in the beginning but removed them and forgot to change the name of the variable.

Reply:
Nice, thanks a lot for the explanations and changes :)